app/templates/model-records/index.hbs

```diff
@@ -1 +1,5 @@
-{{model-records-table recordType=recordType records=model}}
+{{model-records-table
```

Comment:
This thing is weird.  discussed already but still.  just wanted to say it again.